holidays.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
   gem.licenses      = ['MIT']
   gem.required_ruby_version = '~> 2.2'
-  gem.add_development_dependency 'bundler'
-  gem.add_development_dependency 'rake', '~> 12.0'
+  gem.add_development_dependency 'bundler', '~> 1.16'
```

Comment:
Why bundler's version is fixed?

Reply:
The gem build was giving me a warning telling me that it wasn't 'safe' to leave a version off of `bundler`. I added it in mainly to make that warning going away. 😄

Reply:
Oh, i see.:)
Please go ahead :rocket: